packages/webview_flutter/webview_flutter/lib/platform_interface.dart

```diff
@@ -187,6 +188,30 @@ abstract class WebViewPlatformController {
         "WebView loadUrl is not implemented on the current platform");
   }
 
+  /// Loads the specified URL.
+  ///
+  /// [WebViewRequest] must not be null.
```

Comment:
Don't document things that are communicated by the type (and enforced by the compiler)